bin/codefresh-sync.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+## Messages templates
+MESSAGE_SEPARATOR="$(printf -- '_%.0s' {1..80})";
+
+MESSAGE_NO_CHANGES_REQUIRED=">>>>>>>> Processing ${PIPELINE} pipeline ... No changes required";
+
+MESSAGE_CHANGES_REQUIRED=">>>>>>>> Processing ${PIPELINE} pipeline ... Changes found
+${MESSAGE_SEPARATOR}
+These changes will be applied if you run the same command with the environment variable APPLY=true
+or
+make codefresh/sync/apply REPOSITORIES=${REPOSITORY} ACCOUNTS=${ACCOUNT} PIPELINES=${PIPELINE}
+${MESSAGE_SEPARATOR}";
+
+MESSAGE_APPLING_CHANGES=">>>>>>>> Processing ${PIPELINE} pipeline... Changes applied
```

Comment:
Use `\n` here and instead `echo -e $VAR` which will then interpret the `\n`

Reply:
I tried. 
It looks ugly - too long.
It's easy to see what you get with current solution